### ISSUE TITLE
Unsupported Swift version

### ISSUE THREAD

**Issue by goerz** [holzschu/iVim#5]
I had to do a restore on my iPad, which killed iVim. Trying to recompile it results in the following error message in Xcode:

~~~
The targets “iVimTests” and “iVimUITests” contains source code developed with Swift 3.x. This version of Xcode does not support building or migrating Swift 3.x targets.

Use Xcode 10.1 to migrate the code to Swift 4.
~~~

Im currently on Xcode 11.1 on Mojave. I'm not sure how easy it is to downgrade to Xcode 10.1, or if that will allow to do the swift-conversion automatically. Maybe you could commit a converted version of the source code? While running the `get_frameworks.sh` script, some 32-bit warning popped up. I'm not planning to move to Catalina soon, but maybe this would be a good time to make the compilation future-proof in that respect.

You have a version of `python_ios` now that uses Python 3, right (I'm assuming that's what's used in a-Shell now)? I'd definitely prefer to Python 3 over Python 2 in iVim.

**Comment by holzschu** [comment]
Yes, I have the same problem. I am unable to recompile iVim for the time being (that was one of the reasons for starting the work on a-Shell).

**Comment by goerz** [comment]
I see! Too bad, but a-Shell is probably a much better way to move forward in any case.

I'll abuse this comment as a feature suggestion for a-Shell then. If it's going to replace iVim, I think it would ideally have some way to access the "open in place" functionality (through the Share sheet). I could think of two ways:

1. Add Shortcuts support to a-Shell, so that I can have a "run command (with arguments) in a-Shell" as an action in a Shortcuts workflow. I could then create a share-extension shortcut to open a file. I'm not sure if Shortcuts can handle an "open in-place" file as an input, though.

Shortcuts support would be nice to have in any case: I could also create a Vim icon in my dock that opens a-Shell directly in vim, for example.

2. (if Shortcuts can't handle open-in-place) Allow a-Shell as an app to receive files as "open in-place" from the share sheet, and then have a setting in a-Shell for which command to run when it receives the file.

**Comment by holzschu** [comment]
Compilation should work now. The original repository was updated, and so was this one.
Python does not work, but I think I know why.
